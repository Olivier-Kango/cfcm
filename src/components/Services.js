import React from 'react';

const Services = () => (
  <div>
    Nos Services
  </div>
);

export default Services;
