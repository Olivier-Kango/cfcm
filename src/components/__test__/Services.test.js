import { render, screen } from '@testing-library/react';
import Services from '../Services';

describe('Testing the Services page', () => {
  test('should render', () => {
    render(<Services />);
    const about = screen.getByText(/Nos Services/i);
    expect(about).toMatchSnapshot();
  });
});
