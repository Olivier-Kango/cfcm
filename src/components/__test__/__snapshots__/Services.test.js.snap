// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Services page should render 1`] = `
<div>
  Nos Services
</div>
`;
